ios/FluentUI/Vnext/Drawer/MSFSlideOutPanel.swift

```diff
@@ -0,0 +1,259 @@
+    static let initialContentWidthPercent: CGFloat = 0.9
+    static let initialContentHeightPercent: CGFloat = 0.5
+}
+
+/// `MSFSlideOutPanel` in the drawer's  undelrying layer that expands and collapsed on x/y axis of the drawer.
+/// `View` consist of content view placed on top on panel and transparent background layer
+// MARK: - Base Panel
+struct MSFSlideOutPanel<Content: View>: View, MSFPanelContent, MSFPanelTransition {
+
+    /// Only effective when panel is in transition, valid range [0,1]
+    @Binding public var percentTransition: Double?
+
+    /// Configure the apperance of drawer
+    @ObservedObject public var tokens = MSFDrawerTokens()
+
+    /// Content view is visible when slide over panel is expanded
+    var content: Content
+
+    /// Interactive state of panel
+    @Binding var transitionState: MSFDrawerTransitionState
+
+    /// content size set by client
+    var intrinsicContentSize: CGSize?
+
+    /// content size the panel's attempt to set if within bounds
+    var preferredContentSize: CGSize {
+        get {
+            if let contentSize = intrinsicContentSize, contentSize != .zero {
+                return contentSize
+            } else {
+                var preferredWidth = panelSize.width * Constants.initialContentWidthPercent
+                if preferredWidth >= Constants.defaultContentSize.width {
+                    preferredWidth = Constants.defaultContentSize.width
+                }
+
+                var preferredHeight = panelSize.height * Constants.initialContentHeightPercent
+                if preferredHeight >= Constants.defaultContentSize.height {
+                    preferredHeight = Constants.defaultContentSize.height
+                }
+                return CGSize(width: preferredWidth, height: preferredHeight)            }
+        } set {
+            if preferredContentSize != newValue {
+                intrinsicContentSize = newValue
+            }
+        }
+    }
+
+    /// size of the base panel
+    var panelSize: CGSize = UIScreen.main.bounds.size
+
+    /// Action executed with background transperent view is tapped
+    var actionOnBackgroundTap: (() -> Void)?
+
+    /// If set to `true` then token with dimmed bacground value is used
+    var backgroundDimmed: Bool = false
+
+    /// Slide out direction
+    var direction: MSFDrawerDirection = .left
+
+    /// Action executed with transiton state is completed
+    var transitionCompletion: (() -> Void)?
+
+    var body: some View {
+        AnyStack {
+            if direction == .right || direction == .bottom {
+                MSFInteractiveSpacer(defaultBackgroundColor: $tokens.backgroundClearColor)
+                    .onTapGesture (perform: actionOnBackgroundTap ?? {})
+            }
+
+            content
+                .frame(width: contentWidth, height: contentHeight)
+                .offset(x: contentOffset.dx, y: contentOffset.dy)
+                .cornerRadius(Constants.cornerRadius, direction: direction)
```

Comment:
I'm seeing this issue of the corner radius that is only applied on the final drawer state.
Shouldn't it be set directly on the moving panel instead on the placeholder view?


https://user-images.githubusercontent.com/68076145/109073910-e47fd480-76ab-11eb-9996-0a751ae71e23.mov

Reply:
> I'm seeing this issue of the corner radius that is only applied on the final drawer state.
> Shouldn't it be set directly on the moving panel instead on the placeholder view?
> 
>  drawer_vertical_rounded_corner_bug.mov

Thanks for catching this problem, I will update the PR to fix this problem.